sql_schema/create_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE self_compatibility_status (install_name VARCHAR(200), status VARCHAR(20), py_version VARCHAR(5), timestamp TIMESTAMP, details VARCHAR(5000), PRIMARY KEY (install_name, py_version));
```

Comment:
Is there a text type that you can use instead of VARCHAR(5000)? I think that you only need VARCHAR to support queries). Why is py_version 5 characters?